content/about/publications.html

```diff
@@ -22,7 +22,6 @@
 - shen2016discovering
 - Tang2017
 - lee2017
-- Tang2016
```

Comment:
shouldn't this just get moved from preprint to published?

Reply:
Yep, hadn't added that change yet. Just put it in